doorman/src/main/java/io/servicecomb/company/auth/endpoint/rest/AuthenticationController.java

```diff
@@ -35,16 +43,19 @@
     this.authenticationService = authenticationService;
   }
 
-  @RequestMapping("/login")
-  @ResponseBody
-  String login(
+  @RequestMapping(value = "/login", method = POST)
+  ResponseEntity<String> login(
       @RequestParam(USERNAME) String username,
       @RequestParam(PASSWORD) String password) {
 
-    return authenticationService.authenticate(username, password);
+    String token = authenticationService.authenticate(username, password);
```

Comment:
It could be better,  if we can wrap this method(setting the http header) in another utils.